libgraph/src/PropertyGraph.cpp

```diff
@@ -100,33 +99,13 @@ WriteEntityTypeIDsArray(
 
   KATANA_CHECKED(ff->Init());
 
-  if (KATANA_EXPERIMENTAL_ENABLED(UnstableRDGStorageFormat)) {
-    if (entity_type_id_array.size()) {
-      const katana::EntityTypeID* raw = entity_type_id_array.data();
-      auto buf = arrow::Buffer::Wrap(raw, entity_type_id_array.size());
-      arrow::Status aro_sts = ff->Write(buf);
-      if (!aro_sts.ok()) {
-        return katana::ArrowToKatana(aro_sts.code());
-      }
-    }
-  } else {
-    katana::EntityTypeIDArrayHeader data[1] = {
-        {.size = entity_type_id_array.size()}};
-    arrow::Status aro_sts =
-        ff->Write(&data, sizeof(katana::EntityTypeIDArrayHeader));
-
+  if (entity_type_id_array.size()) {
+    const katana::EntityTypeID* raw = entity_type_id_array.data();
+    auto buf = arrow::Buffer::Wrap(raw, entity_type_id_array.size());
+    arrow::Status aro_sts = ff->Write(buf);
```

Comment:
nit: is there any reason why buf is `auto` and the other types are defined explicitly?

Reply:
I don't recall my reasoning originally, but in general when it comes to auto I prefer:
1) don't use auto, as I find it is likely to hide useful information from the programmer. I could figure out what type `entity_type_id_array.data()` using lsp or looking at the method, but reading the declared type is faster. 
2) to use auto only if the type is already clear. eg: `buf` is clearly an `arrow::Buffer`